opm/parser/eclipse/EclipseState/EclipseState.cpp

```diff
@@ -305,6 +309,13 @@ namespace Opm {
         schedule = ScheduleConstPtr( new Schedule(getEclipseGrid() , deck, logger) );
     }
 
+    void EclipseState::initSimulationConfig(DeckConstPtr deck) {
+        auto eqlnum = getIntGridProperty("EQLNUM")->getData();
+        int maxEqlnum = *max_element(eqlnum.begin(), eqlnum.end());
```

Comment:
Write out the `std` in `std::max_element` explicitly - that is the way it is done everywhere else.